lib/warder/coffee_lint_runner/message_line_formatter.rb

```diff
@@ -0,0 +1,31 @@
+module Warder
+  # responsible for run coffeescript style guide validation
+  class CoffeeLintRunner
+    # responsible for human readable error message representation
+    class MessageLineFormatter
+      MESSAGE_KEY = 'message'.freeze
+      CONTEXT_KEY = 'context'.freeze
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.